### ISSUE TITLE
[Feature Request]: Specify non-zero null hypothesis in t test module

### ISSUE THREAD

**Issue by jlwoodard** [jasp-stats/jasp-issues#1887]
### Description

The alternate hypothesis choices for the paired and independent groups situations in the t test module seem to imply a null hypothesis of 0. Some studies may hypothesize that a pre-post difference of at least 5 or more is hypothesized, for example. I see no way to specify this a priori null hypothesis mu

### Purpose

To permit the user to specify a non-zero mu (test value) for the null hypothesis in the paired and independent groups t tests, which is readily available in R code.

### Use-case

t.test(MyData$After, MyData$Before, mu=5, alternative="greater", paired=T)  for the paired t test. A similar specification for mu in the independent groups situation would also be expected.

### Is your feature request related to a problem?

No

### Describe the solution you would like

Offer the user a choice to add a non-zero null hypothesis value (test value) for the paired and independent groups t test.

### Describe alternatives that you have considered

I have looked through the possible specifications in the t test module. I can do the pre-post analysis or test a hypothesized group mean difference using the one-sample t test on the difference scores, but adding a test value option in the paired and independent groups would be helpful. It would not immediately be obvious to most users to approach the analysis using a one-sample test.

**Comment by juliuspf** [comment]
@jlwoodard, thanks for the request.

**Comment by koenderks** [comment]
I believe this would be a simple yet valuable addition, potentially even for the next release, as it can be implemented fairly quickly. @JohnnyDoorn Are you the right person for this (as in maintainer t-test module??

**Comment by JohnnyDoorn** [comment]
Yes, excellent suggestion! I can do this, or perhaps it could serve as a good intro project for @TarandeepKang (since it's adding a single option, where the one-sample t-test already serves as an example)?
